projects/cuelang.org/package.yml

```diff
@@ -0,0 +1,31 @@
+distributable:
+  url: https://github.com/cue-lang/cue/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: cue-lang/cue/tags
+
+provides:
+  - bin/cue
+
+build:
+  script: |
+    go mod download
+    go build -v -ldflags="$LDFLAGS" -o "{{ prefix }}"/bin/cue ./cmd/cue
+  dependencies:
+    go.dev: ^1.18
+  env:
+    LDFLAGS:
+      - -s
+      - -w
+      - -X cuelang.org/go/cmd/cue/cmd.version=v{{ version }}
```

Comment:
@jhheider - I've checked the [documentation on link flags](https://pkg.go.dev/cmd/link) and it doesn't look like there are long-form options.